common/common.go

```diff
@@ -315,7 +315,10 @@ const (
 // Constants used by util/clusterauth package
 const (
 	ClusterAuthRequestTimeout = 10 * time.Second
-	BearerTokenTimeout        = 30 * time.Second
+)
+
+const (
+	BearerTokenTimeout = 30 * time.Second
```

Comment:
This isn't a related change, but when I was looking over things to try to understand patterns, I asked a coworker to look things over with me and we realized that the comment above the block didn't apply to this item, so I'm tossing it in with this change...